rocketpy/motors/LiquidMotor.py

```diff
@@ -179,7 +179,7 @@ def inertiaTensor(self):
         tuple (of Functions)
             Pricipal moment of inertia tensor of the motor, in kg*m^2.
         """
-        self.inertiaI = self.inertiaZ = Function(0)
+        self.inertiaI = self.inertiaZ = 0
```

Comment:
Nice change! This avoids a lambda-defined Function, which unfortunately is *extremely* dominant. By dominant, I mean whenever it interacts if an array-defined Function, it makes the result lambda-defined as well. 

This is a major problem in my point of view in terms of handling constant Functions. This is out of scope, but in another issue/PR, I agree we should discuss an array-dominant Function class implementation, or, a special handling of constant lambda-defined Functions, which are quite common.

Reply:
Indeed that was the rationale behind the change, it took me quite some time to figure out where the things were becoming lambda based and this was the culprit.

Great comment on the Function class dominance, couldn't agree more.